src/test/java/com/facebook/presto/plugin/tiledb/TestTileDBQueries.java

```diff
@@ -1575,7 +1551,7 @@ private void create1DVectorNoAttribute(String arrayName)
     {
         QueryRunner queryRunner = getQueryRunner();
         String createSql = format("CREATE TABLE %s(" +
-                "x bigint WITH (dimension=true) " +
+                "x integer WITH (dimension=true) " +
```

Comment:
Why the change here? Is this from -Java change?

Reply:
Just aligning it with Trino, no particular reason for the change.